modules/rosa-imds-machine-pools-cli.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa_nodes/rosa-managing-worker-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="rosa-imds-machine-pools-cli_{context}"]
+= Enabling Instance Metadata Service on machine pools in CLI
+
+You can select your Instance Metadata Service (IMDS) type when creating your cluster in your ROSA CLI. You can select to use both IMDSv1 and IMDSv2, or you can specify to only use IMDSv2.
+
+.Prerequisites
+
+* You installed and configured the latest AWS (`aws`), ROSA (`rosa`), and OpenShift (`oc`) CLIs on your workstation.
+* You logged in to your Red Hat account by using the `rosa` CLI.
+* You have the permissions to create and manage clusters.
+
+.Procedure
+
+. In your terminal, create a ROSA cluster with your specifications. You must provide a value of either `required` or `optional` for the `--ec2-metadata-http-tokens` flag to enable IMDSv2 or a combination of IMDSv1 and IMDSv2.
++
+[source,terminal]
+----
+$ rosa create cluster --cluster <name_of_cluster> --ec2-metadata-http-tokens <required_or_optional>
+----
+. During the cluster creation process, you see the following line that allows you to specify if you use IMDSv2 only or if you use both IMDSv1 and IMDSv2:
++
+[source,terminal]
+----
+? Configure the use of IMDSv2 for ec2 instances optional/required: required
+----
+
+.Verification
+
+. After your cluster has been created, on your cluster *Overview* tab in {cluster-manager-url}, you see the *Instance Metadata Service (IMDS)* field that notes your IMDS version support.
```

Comment:
Suggestion to reword this as an imperative for users to follow:

```suggestion
. After your cluster has been created, go to your cluster *Overview* tab in {cluster-manager-url} to see the *Instance Metadata Service (IMDS)* field that notes your IMDS version support.
```

Reply:
Or maybe "navigate to" (I see that used a few times elsewhere in the assembly already).